src/reducers/ServiceGraphState.ts

```diff
@@ -0,0 +1,27 @@
+import { ServiceGraphState } from '../store/Store';
```

Comment:
Again, why not all the toggles/state?

Reply:
Yes, that is next. All the toggle buttons via: https://issues.jboss.org/browse/KIALI-654